test/minGov.js

```diff
@@ -210,6 +211,20 @@ contract('MinGov', (accounts) => {
     assert.equal(operator, '0x0000000000000000000000000000000000000000');
   })
 
+  it('should allow to mint token', async () => {
+    const token = await NativeToken.new("Token", "TOK", 18);
+    await token.addMinter(gov.address);
+    // propose and finalize value change
+    const data = await token.contract.methods.addMinter(accounts[1]).encodeABI();
+    await gov.propose(token.address, data);
+    await gov.finalize();
+
+    // test minting rights
+    await token.mint(accounts[1], 100000);
```

Comment:
```suggestion
    await token.mint(accounts[1], 100000, { from: accounts[1] });
```

Reply:
otherwise we are not testing minting rights

Reply:
good point